mailing_list/models.py

```diff
@@ -138,15 +137,26 @@ def get_or_create_or_delete_mailing_lists_for_canvas_course_id(self, canvas_cour
                 mailing_list = MailingList(**create_kwargs)
                 mailing_list.save()
 
+            # cs_class_type is used to determine if the section
+            # is an enrollment section or a non-enrollment section.
+            # if it's null for a section with a real sis section id, we
+            # should consider it an enrollment section.
+            cs_class_type = None
+            if s['sis_section_id'] and s['sis_section_id'].isdigit():
```

Comment:
Do we only need the check for cs_type for those which have s['sis_section_id'].isdigit()?